python/lsst/ip/diffim/detectAndMeasure.py

```diff
@@ -490,6 +525,28 @@ def calculateMetrics(self, difference):
         self.metadata.add("nBadPixelsDetectedPositive", np.sum(detPosPix))
         self.metadata.add("nBadPixelsDetectedNegative", np.sum(detNegPix))
 
+    def _runStreakMasking(self, maskedImage):
+        """Do streak masking at put results into catalog.
+
+        Parameters
+        ----------
+        maskedImage: `lsst.afw.image.maskedImage`
+            The image in which to search for streaks. Must have a detection
+            mask.
+
+        Returns
+        -------
+        streakInfo: `dict` ['str', 'float']
+            Catalog of streak profile information.
+        """
+        streaks = self.maskStreaks.run(maskedImage)
+        rhos = np.array([line.rho for line in streaks.lines])
+        thetas = np.array([line.theta for line in streaks.lines])
+        sigmas = np.array([line.sigma for line in streaks.lines])
+
+        streakInfo = {'rho': rhos, 'theta': thetas, 'sigma': sigmas}
+        return pipeBase.Struct(maskedStreaks=streakInfo)
```

Comment:
I've implemented this so that if `self.config.writeStreakInfo=False`, it returns
```streakInfo = {'rho': np.array([]), 'theta': np.array([]), 'sigma': np.array([])}```
I'm not sure if you meant a totally empty `pipeBase.Struct`, but I did it this way so that the docs are consistent.